openduck-py/openduck_py/routers/voice.py

```diff
@@ -16,23 +17,24 @@
 from openduck_py.voices import styletts2
 from openduck_py.routers.templates import generate
 
-model = whisper.load_model("base.en")  # Fastest possible whisper model
+asr_model = nemo_asr.models.EncDecCTCModelBPE.from_pretrained(model_name="nvidia/stt_en_fastconformer_ctc_large")
 
 audio_router = APIRouter(prefix="/audio")
 
 
 def _transcribe(audio_data):
-    resampled = resample(
-        torch.tensor(audio_data).to("cuda"), orig_freq=24000, new_freq=16000
-    )
-    return model.transcribe(resampled)["text"]
+    with NamedTemporaryFile(suffix=".wav", mode='wb+') as temp_file:
+        wavfile.write(temp_file.name, 16000, audio_data)
+        temp_file.flush()
+        temp_file.seek(0)
+        transcription = asr_model.transcribe([temp_file.name])[0]
```

Comment:
don't we still need the resample? As it's written, we are writing 24khz data to a file with a 16khz header.

Reply:
better yet, could stream 16khz in the first place (unless there's a reason not to)

Reply:
good call, can you make that change in `simple_bot.py`?